README.md

```diff
@@ -1,13 +1,38 @@
 # rskswap-ui
 Project for Swaping RBTC to rWBTC independently from the claim (and the other way round too) 
+
+## Architecture Diagram
+![Architecture Diagram](architecture-diagram.png)
```

Comment:
For the architecture diagram, use the generic rLogin logo instead of metamask. We support not only metamask but all the wallets supported by rLogin